src/main/java/io/icker/factions/ui/InfoGui.java

```diff
@@ -0,0 +1,247 @@
+package io.icker.factions.ui;
+
+import com.mojang.authlib.GameProfile;
+import eu.pb4.sgui.api.elements.GuiElementBuilder;
+import eu.pb4.sgui.api.gui.SimpleGui;
+import io.icker.factions.FactionsMod;
+import io.icker.factions.api.persistents.Faction;
+import io.icker.factions.api.persistents.User;
+import io.icker.factions.util.Command;
+import io.icker.factions.util.GuiInteract;
+import io.icker.factions.util.Icons;
+import net.minecraft.item.Items;
+import net.minecraft.screen.ScreenHandlerType;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.text.Style;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import net.minecraft.util.UserCache;
+import net.minecraft.util.Util;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class InfoGui extends SimpleGui {
+    protected final Runnable closeCallback;
+
+    /**
+     * Constructs a new simple container gui for the supplied player.
+     *
+     * @param player the player to server this gui to
```

Comment:
Not sure what this comment means

Reply:
That's Javadoc from SGui. It's also present in all other GUIs. Should I remove it from everywhere?

Reply:
Unless the comment adds extra information of value that isn't self-evident from skimming the code, I'd remove it. Personally, I don't think this comment is needed.